.travis.yml

```diff
@@ -289,6 +289,7 @@ install:
               return
           fi
           phpenv global ${PHP/hhvm*/hhvm}
+          composer self-update
```

Comment:
What's about updating composer before using it? (`composer global require` at line 270)

Reply:
Travis has one composer per php version. We must update it after switching the version of PHP.
Using composer 1 for simple things (like L270) makes zero diff with c2, no need to care more IMHO.

Reply:
Isn-t the syntax composer self-update --2`